docs/src/whatsnew/latest.rst

```diff
@@ -48,8 +48,8 @@ This document explains the changes made to Iris for this release
 🚀 Performance Enhancements
 ===========================
 
-#. N/A
-
+#. `@bouweandela`_ made the time coordinate categorisation functions in
+   :mod:`iris.coord_categorisation` faster. (:pull:`5999`)
```

Comment:
`add_categorised_coord()` is public API because it also allows user custom categorisation. It would be good to make clear that this improvement is available to anyone using `add_categorised_coord()` with `cftime.datetime` .

Reply:
Done in 16b20b3